src/config.rs

```diff
@@ -282,6 +297,13 @@ impl Config {
         config.include_idle = matches.occurrences_of("idle") > 0;
         config.gil_only = matches.occurrences_of("gil") > 0;
         config.include_thread_ids = matches.occurrences_of("threads") > 0;
+        #[cfg(all(target_os = "linux", feature = "trace_syscalls"))]
+        {
+            config.trace_syscalls = matches.occurrences_of("trace_syscalls") > 0;
+            if config.trace_syscalls {
+                config.include_idle = true;
+            }
```

Comment:
Why do we need to include idle frames when we're including syscalls?

Reply:
That's a good question: if you call `time.sleep(1)` python will effectively run the `select()` syscall. What should happen? Should py-spy show the select syscall or should it ignore the thread because effectively it's idle?
Maybe you are right that this decision should be taken care of by the user, so --syscalls should not include --idle (only if the user specifies this argument).
I just wanted to bring the problem with `time.sleep()` to your attention, but probably --syscalls should not necessarily inlcude --idle

Reply:
I changed this, --syscalls does not imply --idle